src/components/widget/RawWidgetHelpers.js

```diff
@@ -39,7 +39,17 @@ export function generateMomentObj(value, FORMAT) {
 }
 
 export function getClassNames({ icon, forcedPrimary } = {}) {
-  const { widgetData, gridAlign, type, updated, rowId, isModal } = this.props;
+  const {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `⏎····getWidgetData,⏎····gridAlign,⏎····type,⏎····updated,⏎····rowId,⏎····isModal,⏎··}·=·this.props;` with `·getWidgetData,·gridAlign,·type,·updated,·rowId,·isModal·}·=·this.props`](https://app.codacy.com/manual/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4779808)